src/summary.ts

```diff
@@ -150,14 +151,21 @@ export function addLicensesToSummary(
   }
 }
 
-function getManifests(changes: Changes): Set<string> {
-  return new Set(changes.flatMap(c => c.manifest))
-}
+export function addScannedDependencies(changes: Changes): void {
+  const dependencies = groupDependenciesByManifest(changes)
+  const manifests = dependencies.keys()
 
-function renderUrl(url: string | null, text: string): string {
-  if (url) {
-    return `<a href="${url}">${text}</a>`
-  } else {
-    return text
+  const summary = core.summary
+    .addHeading('Scanned Dependencies')
+    .addRaw(`We scanned ${dependencies.size} manifest files:`)
+
+  for (const manifest of manifests) {
+    const deps = dependencies.get(manifest)
+    if (deps) {
+      const dependencyNames = deps.map(
+        dependency => `<li>${dependency.name}@${dependency.version}</li>`
```

Comment:
Don't love creating these by hands, but as far as I can tell the [Actions Summary JS API](https://github.com/actions/toolkit/blob/main/packages/core/src/summary.ts) is not publicly documented (?), and I couldn't find a way in there to create nested lists inside a loop.